subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/internal/ValidateAction.java

```diff
@@ -73,7 +73,7 @@ public interface Params extends WorkParameters {
 
     }
 
-//    @Inject
+    //    @Inject
 //    protected abstract Problems getProblemsService();
```

Comment:
This seems like an accidental change.
Should we remove commented lines?

Reply:
already in the works on the next PR :)